custom_components/checkwatt/__init__.py

```diff
@@ -128,10 +128,13 @@ async def _async_update_data(self) -> CheckwattResp:
 
             async with CheckwattManager(username, password) as cw_inst:
                 if not await cw_inst.login():
-                    raise InvalidAuth
+                    _LOGGER.error("Failed to login, abort update")
+                    raise UpdateFailed("Failed to login")
```

Comment:
LoginFailed?
Maybe it is easier to use same raise error?

Reply:
...Ill keep it like this for now. I do not know the consequences for raising that error